content/en/synthetics/api_tests/_index.md

```diff
@@ -0,0 +1,179 @@
+
+{{% tab "HTTP Test" %}}
+
+Define the HTTP request you want to be executed by Datadog:
+
+{{< img src="synthetics/api_tests/make-http-request.png" alt="Make HTTP Request" responsive="true" style="width:80%;" >}}
+
+1. Choose the `Method` and `URL` to query. Available methods are:
+   - GET
+   - POST
+   - PATCH
+   - PUT
+   - DELETE
+   - OPTIONS
+2. Optionally, click on **Advanced options** to enrich your request with custom request headers, authentication credentials, body content, or cookies. Toggle **Follow redirects** to have the monitored endpoint follow up to ten redirects.
+3. Name your HTTP test.
+4. Optionally, add tags to filter your HTTP test in the [Synthetics page][1].
+5. Pick the locations to run the test from. Many AWS locations from around the world are available, the full list can be retrieved [through Datadog API][2].
+6. Choose a check frequency between "1 run per minute" and "1 run per week".
+7. Click on **Test URL** to try out the request configuration. You should see a response preview show up on the right side of your screen.
+
+[1]: /synthetics
+[2]: /api/?lang=bash#get-available-locations
+{{% /tab %}}
+
+{{% tab "SSL Test" %}}
+
+Define the SSL request you want to be executed by Datadog:
+
+{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Make SSL Request" responsive="true" style="width:80%;" >}}
+
+1. Specify the `Host` and the SSL `Port`. By default, it is set to _443_.
+2. Name your SSL test.
+3. Optionally, add tags to filter your SSL test in the [Synthetics page][1].
+4. Pick the locations to run the test from. Many AWS locations from around the world are available, the full list can be retrieved [through Datadog API][2].
+5. Choose a check frequency between "1 run per minute" and "1 run per week".
+6. Click on **Test Connection** to try out the request configuration. You should see a response preview show up on the right side of your screen.
+
+[1]: /synthetics
+[2]: /api/?lang=bash#get-available-locations
+{{% /tab %}}
+
+{{< /tabs >}}
+
+### Alert Conditions
+
+Set alert conditions to determine the circumstances under which you want a test to send an alert. When you set the alert conditions to: `An alert is triggered if any assertion fails for X minutes from any n of N locations`, an alert is triggered if:
+
+* At least one location was in failure (at least one assertion failed) during the last *X* minutes, **AND**
+* At one moment during the last *X* minutes, at least *n* locations were in failure
+
+The uptime bar is displayed differently on your test result: location uptime is displayed on a per-evaluation basis (whether the last test was up or down). Total uptime is displayed based on the configured alert conditions. Notifications sent are based on the total uptime bar.
+
+#### Assertions
+
+When running an API test, you must define at least one assertion that should be monitored by Datadog.
+An assertion is defined by a parameter, an optional property, a comparator and a target value.
```

Comment:
```suggestion
An assertion is defined by a parameter, an optional property, a comparator, and a target value.
```

Reply:
In French it is a syntax error, but in english it's the right way to do ^^